src/main/resources/META-INF/plugin.xml

```diff
@@ -104,6 +104,14 @@
             key="inspection.not.null.checkable.property.source.used.with.default.property"
             shortName="NotNullCheckableSourcePropertyUsedWithDefaultValue"
             implementationClass="org.mapstruct.intellij.inspection.NotNullCheckableSourcePropertyUsedWithDefaultValueInspection"/>
+    <localInspection
+            language="JAVA"
+            enabledByDefault="true"
+            level="WARNING"
+            bundle="org.mapstruct.intellij.messages.MapStructBundle"
+            key="inspection.java.expression.unnecessary.whitespace"
```

Comment:
Just had a look at the settings page. There is no other inspection title containing `{0}` or `{1}`.
Therefore I would suggest another key only for the title: `inspection.java.expression.unnecessary.whitespace.title=Unnecessary whitespaces before Java expression`

Reply:
I added the title property.